src/data/showcases.js

```diff
@@ -1235,6 +1235,15 @@ export const Showcases = [
     source: null,
     tags: ["analytics"],
   },
+  {
+    title: "Langarnia",
+    description:
+      "Langarnia is a 3D fantasy game designed to teach languages.",
+    preview: require("./showcase/langarnia.png"),
+    website: "https://www.langarnia.com",
+    source: null,
+    tags: ["educational", "game"],
```

Comment:
```suggestion
    tags: ["game"],
```
The `educational` tag is for education _about Cardano_.  As it says in the `showcases.js` file you are amending:
> Educational projects that will help you onboarding to Cardano.